File: 1036.c
#include <math.h>
#include <stdio.h>

typedef struct 
{
    float x1, x2;
} Raiz;

Raiz calculaRaiz(float, float, float);

int main(void)
{
    float a, b, c;
    scanf("%f %f %f", &a, &b, &c);

    float delta = b*b - 4 * a * c;
    if (delta < 0 || a == 0)
    {
        printf("Impossivel calcular\n");
        return 0;
    }
    
    Raiz resultado = calculaRaiz(a, b, c);

    printf("R1 = %.5f\nR2 = %.5f\n", resultado.x1, resultado.x2);

    return 0;
}

Raiz calculaRaiz(float a, float b, float c)
{
    float delta = b*b - 4 * a * c;
    Raiz retorno = {(-b + sqrt(delta))/(a*2),
                    (-b - sqrt(delta))/(a*2)};
    return retorno;
}

File: beecrowd/1036.py
#!python3.8

from math import pow, sqrt

values = [float(x) for x in input().split()]

delta = pow(values[1], 2) - 4 * values[0] * values[2]

if delta < 0 or values[0] == 0:
    print('Impossivel calcular')

else:
    roots = [
        ((-values[1]) + sqrt(delta))/(2 * values[0]),
        ((-values[1]) - sqrt(delta))/(2 * values[0])
    ]
    # python3.8
    # print(f'R1 = {roots[0]:.5f}\nR2 = {roots[1]:.5f}')
    # python3.4
    print('R1 = {:.5f}\nR2 = {:.5f}' .format(roots[0], roots[1]))
